Makefile

```diff
@@ -103,6 +104,12 @@ build-tar: ./bin/node-problem-detector ./bin/log-counter
 
 build: build-container build-tar
 
+docker-builder:
+	docker build -t npd-builder ./builder
+
+build-in-docker: clean docker-builder
```

Comment:
ditto. Add `build-in-docker` to the `.PHONY` dummy target dependency list.

Reply:
Done.